### ISSUE TITLE
Name Expiration Warning Plugin

### ISSUE THREAD

**Issue by phelixbtc** [namecoin/nmcontrol#65]
Have a notification pop up if a name is close to expiry. Could also go into the client GUI

**Comment by JeremyRand** [comment]
How would NMControl choose which names to show notifications for?  I think this would be better to have in Namecoin Core since it's most easily implemented with access to the wallet.

**Comment by phelixbtc** [comment]
It could simply sneak a peek. :)  (name_list)  Or you could manually set names to watch. I suggest to leave this issue open here until we have settled on a new GUI.

**Comment by JeremyRand** [comment]
@phelixbtc Fair enough.  Can we put the low priority label on this one until that point?
